docs/examples/example_Style.qml

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2023 MapLibre contributors
```

Comment:
2023?

Reply:
Indeed I need to update copyright headers in those files to match what we promise people. This repo is a bit messy as we inherited some old files from Mapbox/Qt.